README.md

```diff
@@ -1,463 +1,475 @@
 
-It's _so_ easy!
+## Installation
 
-`npm install tough-cookie`
+It's _so_ easy! Install with `npm` or your preferred package manager.
 
-Why the name?  NPM modules `cookie`, `cookies` and `cookiejar` were already taken.
-
-## Version Support
+```sh
+npm install tough-cookie
+```
 
-Support for versions of node.js will follow that of the [request](https://www.npmjs.com/package/request) module.
+## Node.js Version Support
 
-# API
+Inspired by [AVA's support policy](https://github.com/avajs/ava/blob/v4.0.0/docs/support-statement.md), we follow the [node.js release schedule](https://github.com/nodejs/Release#release-schedule) and support versions that are in Active LTS or Maintenance. We only drop support for older versions of node when we do a major release.
 
-## tough
+## API
 
-Functions on the module you get from `require('tough-cookie')`.  All can be used as pure functions and don't need to be "bound".
+### tough
 
-**Note**: prior to 1.0.x, several of these functions took a `strict` parameter. This has since been removed from the API as it was no longer necessary.
+The top-level exports from `require('tough-cookie')`. All can be used as pure functions and don't need to be bound.
 
-### `parseDate(string)`
+#### `parseDate(string)`
 
-Parse a cookie date string into a `Date`.  Parses according to RFC6265 Section 5.1.1, not `Date.parse()`.
+Parse a cookie date string into a `Date`. Parses according to RFC 6265 Section 5.1.1, not `Date.parse()`.
 
-### `formatDate(date)`
+#### `formatDate(date)`
 
-Format a Date into a RFC1123 string (the RFC6265-recommended format).
+Format a `Date` into an RFC 1123 string (the RFC 6265 recommended format).
 
-### `canonicalDomain(str)`
+#### `canonicalDomain(str)`
 
-Transforms a domain-name into a canonical domain-name.  The canonical domain-name is a trimmed, lowercased, stripped-of-leading-dot and optionally punycode-encoded domain-name (Section 5.1.2 of RFC6265).  For the most part, this function is idempotent (can be run again on its output without ill effects).
+Transforms a domain name into a canonical domain name. The canonical domain name is a domain name that has been trimmed, lowercased, stripped of leading dot, and optionally punycode-encoded (Section 5.1.2 of RFC 6265). For the most part, this function is idempotent (calling the function with the output from a previous call returns the same output).
 
-### `domainMatch(str,domStr[,canonicalize=true])`
+#### `domainMatch(str, domStr[, canonicalize=true])`
 
-Answers "does this real domain match the domain in a cookie?".  The `str` is the "current" domain-name and the `domStr` is the "cookie" domain-name.  Matches according to RFC6265 Section 5.1.3, but it helps to think of it as a "suffix match".
+Answers "does this real domain match the domain in a cookie?". The `str` is the "current" domain name and the `domStr` is the "cookie" domain name. Matches according to RFC 6265 Section 5.1.3, but it helps to think of it as a "suffix match".
```

Comment:
Include link to [Section 5.1.3](https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.3) of the RFC